exercises/word-count/uWordCountTests.pas

```diff
@@ -0,0 +1,244 @@
+unit uWordCountTests;
+
+interface
+uses
+  System.Generics.Collections, DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  WordCountTests = class(TObject)
+  private
+    procedure CompareDictionaries(Expected, Actual: TDictionary<String, integer>);
+  public
+    [Test]
+    procedure Validate_CompareDictionaries;
+	
+    [Test]
+    [Ignore]
```

Comment:
@amoradell Validate_CompareDictionaries is just an "internal" test to show that CompareDictionaries is working as expected.

`Count_one_word` is the first case of the exercise.  It is traditional that the first exercise not be ignored.  Instead I comment the `[Ignore]` attribute but also include along with it a blurb that explains to the student that they will need to comment the other `[Ignore]` attributes when they are ready for those cases to execute.  For this instance `Count_one_word` should look like:

```pascal
    [Test]
//    [Ignore('Comment the "[Ignore]" statement to run the test')]
    procedure Count_one_word;
```